app/views/shipit/stacks/_banners.html.erb

```diff
@@ -36,7 +36,10 @@
       <div class="banner__content">
         <h2 class="banner__title">
           <i class="icon icon--lock"></i>
-          Deploys are locked by <%= stack.lock_author.name %>
+          Deploys were locked by <%= stack.lock_author.name %>
+          <% unless stack.locked_since.nil? %>
+            <%= time_ago_in_words(stack.locked_since) %> ago
```

Comment:
Actually I just noticed this will be slightly bugged. 

It will be on `Stack#show` which is heavily cached.

Luckily we have a helper for this purpose that use jquery.timeago. Let me find it.

Reply:
It's `timeago_tag`: https://github.com/Shopify/shipit-engine/blob/eba7226f04cf16f55fcaa0a89696aed8ee287974/app/views/shipit/tasks/show.html.erb#L3

Once you've changed that :shipit: from me.

Reply:
Good catch. Fixing